spec/20-http_request_header_format.md

```diff
@@ -83,9 +83,7 @@ The dash (`-`) character is used as a delimiter between fields.
 version         = 2HEXDIGLC   ; this document assumes version 00. Version ff is forbidden
 ```
 
-The value is US-ASCII encoded (which is UTF-8 compliant).
-
-Version (`version`) is 1 byte representing an 8-bit unsigned integer. Version `ff` is invalid. The current specification assumes the `version` is set to `00`.
+Version (`version`) is an 8-bit unsigned integer value, serialized as an ASCII string with two characters. Version 255 (`"ff"`) is invalid. This specification assumes the version is set to 0 (`"00"`).
```

Comment:
To avoid ambiguity, it may be worth calling out that this is an ASCII case insensitive string (with a link to the corresponding spec), similar to what you have added in PR #513.

Reply:
It isn't, though. We specify 
* `version = 2HEXDIGLC ` and
* `HEXDIGLC = DIGIT / "a" / "b" / "c" / "d" / "e" / "f" ; lowercase hex character`

So upper case letters `A` to `F` are invalid, aren't they?

Reply:
Thanks, I missed that earlier; in that case, the current wording sounds good to me.